app-misc/fhem/fhem-5.8.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="A GPL'd perl server for house automation"
+HOMEPAGE="https://www.fhem.de/"
+SRC_URI="https://www.fhem.de/${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="dev-perl/Crypt-CBC
+	dev-perl/Device-SerialPort
+	dev-perl/Digest-CRC
+	dev-perl/JSON"
+
+DEPEND="media-gfx/pngcrush"
+
+S="${WORKDIR}/${P}"
```

Comment:
workdir is wrong, see here why: https://devmanual.gentoo.org/ebuild-writing/variables/index.html

Reply:
Yes, since it's the default, I've removed it. Guess, I've forgotten it.